asyncnsq/tcp/connection.py

```diff
@@ -266,4 +266,11 @@ def _finish_upgrading(self, resp=None):
         self._is_upgrading = False
 
     def __repr__(self):
-        return '<NsqConnection: {}:{}'.format(self._host, self._port)
+        return '<TcpConnection: {}:{}>'.format(self._host, self._port)
+
+    def is_starved(self, rdy_count):
+        if self._queue.qsize():
+            starved = False
+        else:
+            starved = (self.in_flight > 0 and self.in_flight >= (rdy_count * 0.85))
+        return starved
```

Comment:
@aohan237 this method has been removed in 320faea08c098dcd76d78c9807c64266a490b553 but it is still in use, see:  https://github.com/aohan237/asyncnsq/blob/35aaca2cc80995b2a3afde6f9bd5d930196db98a/asyncnsq/tcp/reader.py#L164-L166

So I took the liberty to revert it back with a little improvement. I wonder what would you think?

Reply:
is_starved should be deleted.
because when reader begin to work, its starving state is maintained by rdy control, and as the official nsq doc said , client will follow the rule to update rdy state automatically.
so in user scene, the is_starved state should not be used in any cases